static/src/javascripts/projects/common/modules/article/space-filler.js

```diff
@@ -1,66 +1,47 @@
-/* jscs:disable disallowDanglingUnderscores */
 define([
-    'common/utils/$',
-    'common/utils/fastdom-idle',
+    'common/utils/fastdom-promise',
     'Promise',
     'raven',
     'common/modules/article/spacefinder'
 ], function (
-    $,
-    idleFastdom,
+    fastdom,
```

Comment:
We're calling the module fastdom, but is that appropriate any longer? Increasingly we're actually building a Guardian-flavoured proxy for non-immediate DOM writes, and using the proxy as a hook for various optimizations. I think the name 'fastdom' for this module gives us the worst of both worlds: it constrains us not to diverge too radically from the underlying implementation, yet it also misleads readers. 

Instead I would be bold and embrace the fact we're building a specialized DOM proxy for the Guardian's distinct requirements. To that end, I might call it something like `lazyDom` instead. What do you think?